src/app/containers/ArticleMain/index.test.jsx

```diff
@@ -61,23 +71,45 @@ it('should use headline for meta description if summary does not exist', async (
   ).toEqual('Article Headline for SEO');
 });
 
-shouldMatchSnapshot(
-  'should render a news article correctly',
-  <Context service="news">
-    <ArticleMain articleData={articleDataNews} />
-  </Context>,
-);
+it('should render a news article correctly', async () => {
+  fetch.mockResponse(JSON.stringify(newsMostReadData));
 
-shouldMatchSnapshot(
-  'should render a persian article correctly',
-  <Context service="persian">
-    <ArticleMain articleData={articleDataPersian} />
-  </Context>,
-);
+  const { container } = render(
+    <Context service="news">
+      <ArticleMain articleData={articleDataNews} />
+    </Context>,
+  );
 
-shouldMatchSnapshot(
-  'should render a pidgin article correctly (with navigation)',
-  <Context service="pidgin">
-    <ArticleMain articleData={articleDataPidgin} />
-  </Context>,
-);
+  await waitForElement(() => container.querySelector('#Most-Read'));
+
+  expect(container).toMatchSnapshot();
+});
+
+it('should render a persian article correctly', async () => {
+  fetch.mockResponse(JSON.stringify(persianMostReadData));
+
+  const { container } = render(
+    <Context service="persian">
+      <ArticleMain articleData={articleDataPersian} />
+    </Context>,
+  );
+
+  await waitForElement(() => container.querySelector('#Most-Read'));
+
+  expect(container).toMatchSnapshot();
+});
+
+// TODO: Is this change removing navigation?
+it('should render a pidgin article correctly (with navigation)', async () => {
```

Comment:
```suggestion
it('should render a pidgin article correctly (with navigation)', async () => {
```
Just wondering - why is `(with navigation)` pointed out here? I believe all of the pidgin/news/persian should have navigation?

Reply:
It was part of the existing snapshot description, I was going to do a bit of asking around on this one. Also, `shouldMatchSnapshot` tests the snapshot inside the complete html page, while this snapshot just tests the container html itself, but allows us to do the wait 
```js
await waitForElement(() => container.querySelector('#Most-Read'));
```
I'm not sure if this is an issue though? What were the reasons for wanting to render the complete page html? And maybe now I'm missing the navigation, but I need to do some digging to see if it actually was in the original snapshot.